### PR TITLE
Add intro to TDD event

### PR DESCRIPTION
**Pull request by saragibby**
@jewlofthelotus I called this event "Intro to TDD". But the title on Meetup is "TDD or: How I Learned to Stop Worrying and Love Tests". Should I change the slides to match Meetup?

### CODE REVIEW

**Comment by jewlofthelotus** [comment]
Yeah, let's go with the full title - I'd just play around with using `small` tags and color classes to get it looking ok. Check out one of the recent event intros to see how I did it when it was a upcoming event.

File: events/0515_tdd.html [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+                <p class="box copy--small"><small><strong>Girl Develop It is dedicated to providing a harrasment free learning experience for <em>everyone</em>.</strong><br/>For more information, see our <a href="http://www.girldevelopit.com/code-of-conduct" target="_blank">Code of Conduct</a>.</small></p>
+            </section>
+
+            <!-- Upcoming Events -->
+            <section>
+                <h3>Upcoming Events</h3>
+                <div class="box">
+                    <h3 class="event__title"><span class="blue">1-year Anniversary Fundraiser</span></h3>
+                    <h3 class="event__title"><small>Thursday 6/4 @ The Forge (Pillar)</small></h3>
+                    <p><small>6pm - 9pm &nbsp; | &nbsp; 301 E Liberty St, Ann Arbor</small></p>
```
Comment (jewlofthelotus) [inline events/0515_tdd.html]: can you add the http://bit.ly/gdia2-anniversary link to the bottom of this slide

File: events/0515_tdd.html [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+            <section>
+                <h3>Partners!</h3>
+                <div style="background: #fff; padding-top: 80px;" class="box--small">
+                    <img src="images/sponsors-2015.2.png" class="img--bare" />
+                </div>
+            </section>
+
+            <!-- Sponsor Spotlight -->
+            <section>
+                <h3>Sponsor Spotlight!</h3>
```
Comment (jewlofthelotus) [inline events/0515_tdd.html]: we can get rid of AO, no one is ever there

**Comment by jewlofthelotus** [comment]
aside from those comments, looks good. the only other thing is to add the link to this page on `events.html`

**Comment by saragibby** [comment]
Thanks @jewlofthelotus!! good catches!
Made the changes, let me know if you see anything else.

**Comment by jewlofthelotus** [comment]
Hmmm just pulled this up on my laptop and the bottom is getting cut off the title slides... can we move all the social stuff to one line and maybe use `box--small` instead of `box`. I think that'll be it.

**Comment by saragibby** [comment]
yikes, I totally missed that. fixing now

**Pull request merged**